cypress/page-objects/homePageMenu/MenuItems.js

```diff
@@ -0,0 +1,11 @@
+export default class MenuItems {
```

Comment:
not sure about `default` it does not actually default
please remove it because in the next exercise you have some inconsistency